packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -1,48 +1,57 @@
 <ui5-dialog
+	dir="{{effectiveDir}}"
 	aria-label="{{_dialogTitle}}"
 	@ui5-after-open={{_focusRecentlyUsedControl}}
 	@ui5-before-close={{_restoreConfirmedOnEscape}}
 	?stretch={{_isPhone}}
 >
 
 	<div slot="header" class="ui5-vsd-header">
-		<ui5-bar>
-			<div slot="startContent" class="ui5-vsd-start">
+		<div class="ui5-vsd-header-container">
+			<ui5-bar class="ui5-vsd-bar-header">
 				{{#if showBackButton}}
 					<ui5-button
+						slot="startContent"
 						design="Transparent"
 						icon="nav-back"
 						class="ui5-vsd-back-button"
 						@click="{{_navigateToFilters}}"
 					></ui5-button>
-					<ui5-label>{{_filterByTitle}}</ui5-label>
+					<ui5-label slot="startContent" class="ui5-vsd-title">{{_filterByTitle}}</ui5-label>
 				{{else}}
-					<div class="ui5-vsd-bar-title">
-						<ui5-label class="ui5-vsd-title">{{_dialogTitle}}</ui5-label>
-						<ui5-segmented-button @ui5-selection-change="{{_handleModeChange}}">
-							<ui5-segmented-button-item
-								?pressed="{{isModeSort}}"
-								?focused="{{isModeSort}}"
-								icon="sort"
-								mode="Sort"
-							></ui5-segmented-button-item>
-							<ui5-segmented-button-item
-								?pressed="{{isModeFilter}}"
-								?focused="{{isModeFilter}}"
-								icon="filter"
-								mode="Filter"
-							></ui5-segmented-button-item>
-						</ui5-segmented-button>
-					</div>
+					<ui5-label slot="startContent" class="ui5-vsd-title">{{_dialogTitle}}</ui5-label>
 				{{/if}}
+
+				<ui5-button
+					slot="endContent"
+					design="Transparent"
+					@click="{{_resetSettings}}"
+					?disabled={{_disableResetButton}}
+					reset-button>
+						{{_resetButtonLabel}}
+				</ui5-button>
+			</ui5-bar>
+		</div>
+		{{#unless showBackButton}}
+			<div class="ui5-vsd-sub-header-container">
+				<ui5-bar class="ui5-vsd-bar-sub-header">
```

Comment:
You can add the segmented button in the first bar & use only one bar at all

Reply:
I don't think that's possible

Reply:
It is possible. We can comment the approach if you have any questions